flexget/plugins/filter/translations.py

```diff
@@ -0,0 +1,594 @@
+                                        'enum': [ACTION_ACCEPT, ACTION_REJECT, ACTION_DO_NOTHING],
+                                    },
+                                    DEFAULT: {
+                                        'type': 'string',
+                                        'enum': [ACTION_ACCEPT, ACTION_REJECT, ACTION_DO_NOTHING],
+                                    },
+                                    UNKNOWN: {
+                                        'type': 'string',
+                                        'enum': [ACTION_ACCEPT, ACTION_REJECT, ACTION_DO_NOTHING],
+                                    },
+                                },
+                                'additionalProperties': {
+                                    'type': 'string',
+                                    'enum': [ACTION_ACCEPT, ACTION_REJECT, ACTION_DO_NOTHING],
+                                },
+                            },
+                        ]
+                    },
+                    'subbed': {
+                        'oneOf': [
+                            {
+                                "type": 'string',
+                                'enum': [ACTION_ACCEPT, ACTION_REJECT, ACTION_DO_NOTHING],
+                            },
+                            {
+                                "type": 'object',
+                                'properties': {
+                                    OTHER: {
+                                        'type': 'string',
+                                        'enum': [ACTION_ACCEPT, ACTION_REJECT, ACTION_DO_NOTHING],
+                                    },
+                                    NONE: {
+                                        'type': 'string',
+                                        'enum': [ACTION_ACCEPT, ACTION_REJECT, ACTION_DO_NOTHING],
+                                    },
+                                    DEFAULT: {
+                                        'type': 'string',
+                                        'enum': [ACTION_ACCEPT, ACTION_REJECT, ACTION_DO_NOTHING],
+                                    },
+                                    UNKNOWN: {
+                                        'type': 'string',
+                                        'enum': [ACTION_ACCEPT, ACTION_REJECT, ACTION_DO_NOTHING],
+                                    },
+                                },
+                                'additionalProperties': {
+                                    'type': 'string',
+                                    'enum': [ACTION_ACCEPT, ACTION_REJECT, ACTION_DO_NOTHING],
+                                },
+                            },
+                        ]
+                    },
+                },
+            },
+        ]
+    }
+
+    def clean_symbols(self, text: str) -> str:
+        """Replaces common symbols with spaces. Also normalize unicode strings in decomposed form.
+
+        Args:
```

Comment:
reStructured would be preferable, although not as critical anymore with type hinting